background/contextmenu.js

```diff
@@ -27,19 +27,23 @@ browser.contextMenus.create({
   'title': browser.i18n.getMessage('contextFillUser'),
   'contexts': ['editable']
 });
-browser.runtime.getBrowserInfo().then((info) => {
-  let ctx;
-  if (Number.parseInt(info.version.split('.')[0], 10) >= 53) {
-    ctx = 'password';
-  } else {
-    ctx = 'editable';
-  }
+
+try {
```

Comment:
This is better done with an explicit check instead of try-catch.
Also, this looks like it will break FF by not creating the context menu, since there is no call to `create` if there is no exception

Reply:
Another option is to have an extra js file that guards compatibility in addition to the polyfill, such that a manual `browser.runtime.getBrowserInfo()` could be added